website/docs/r/cosmosdb_mongo_database.html.markdown

```diff
@@ -36,6 +37,8 @@ The following arguments are supported:
 
 * `account_name` - (Required) The name of the Cosmos DB Mongo Database to create the table within. Changing this forces a new resource to be created.
 
+* `throughput` - (Optional) The throughput of the MongoDB collection (RU/s). Must be set in increments of `100`. The minimum value is `400`. This must be set upon database creation otherwise it cannot be updated without a manual terraform destroy-apply.
```

Comment:
Could we make this
```suggestion
* `throughput` - (Optional) The throughput of the MongoDB collection (RU/s). Must be set in increments of `100`, and defaults to `400`.

-> **NOTE:** The minimum value is `400`. This must be set upon database creation otherwise it cannot be updated without a manual terraform destroy-apply.
```